### ISSUE TITLE
Symfony 5.0 support

### ISSUE THREAD

**Issue by patpilus** [shopware/shopware#303]
**Description**  
As Symfony 5.0.0 was released (https://symfony.com/blog/symfony-5-0-0-released) please consider upgrading to the new major version. 

Currently Symfony components are locked at `4.3.8` version (eg. https://github.com/shopware/platform/blob/master/composer.json#L65) in `composer.json` so it's impossible to update to 5.0 or even 4.4 LTS.

**Comment by keulinho** [comment]
We are already working on Symfony 4.4 and Symfony 5.0 support, but we probably have to wait for some external dependencies until they add support for the new Symfony versions, especially 5.0.

We will update to 4.4 as soon as possible, when we`re going to update to 5.0 is not only a question of being technically able to do it, but is also a strategically question, as the 4.4 version will provide LTS.

TLDR;
We are working on it ;)

**Comment by keulinho** [comment]
Yes you may have already seen we've updated our symfony dependencies to 4.4.0

**Comment by patpilus** [comment]
@keulinho thanks 👍I hope that's not the end and soon we will see 5.0 support on the horizon :)

**Comment by shyim** [comment]
Just as update with 6.4.0.0 we will update to newest Symfony version (current 5.2)

**Comment by AndreasA** [comment]
Any news regarding this because the composer.json in the current `trunk` branch (Shopware 6.4.x-dev) still uses Symfoyn 4.4 but I guess this might still be added before 6.4 release.

**Comment by mitelg** [comment]
the Symfony update is currently work in progress 💪

**Comment by shyim** [comment]
Is now in trunk
